### PR TITLE
Heroku - review and corrections

### PR DESCRIPTION
**Pull request by mihaitusa**
Signed-off-by: Mihail-Cristian Tusa <mihail-cristian.tusa@hpe.com

### CODE REVIEW

**Comment by mihaitusa** [comment]
closes #472

**Comment by mihaitusa** [comment]
@gituser173  Please review it!

File: content/io/cloudslang/paas/heroku/account/update_account.sl [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+  workflow:
+    - add_allow_tracking_value:
+        do:
+          json.add_value:
+            - json_input: "{}"
+            - json_path: ['allow_tracking']
+            - value: ${bool(allow_tracking)}
+        publish:
+          - json_output
```
Comment (gituser173) [inline content/io/cloudslang/paas/heroku/account/update_account.sl]: maybe rename json_output to update_account_json as it is used throughout the update_account flow? WDYT?

File: content/io/cloudslang/paas/heroku/account/update_account_email.sl [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        publish:
+          - json_output
+          - return_result
+          - error_message
+          - return_code
+        navigate:
+          SUCCESS: add_password_value
+          FAILURE: ADD_EMAIL_FAILURE
+
+    - validate_password_input:
```
Comment (gituser173) [inline content/io/cloudslang/paas/heroku/account/update_account_email.sl]: why not make password input required: true?

File: content/io/cloudslang/paas/heroku/addons/list_addons_for_application.sl [hunk 3]
```diff
@@ -0,0 +1,47 @@
+#################################################### 
+# This flow performs an REST API call in order to list all existing Heroku add-ons for <app_id_or_name> application
```
Comment (gituser173) [inline content/io/cloudslang/paas/heroku/addons/list_addons_for_application.sl]: typo: "an REST API" should be "a REST API"

**Comment by gituser173** [comment]
Looks good :+1:

**Pull request merged**